nerdlets/pathpoint-nerdlet/services/DataManager.js

```diff
@@ -2826,6 +2873,7 @@ export default class DataManager {
                   query_start: '',
                   query_body: measure.query,
                   query_footer: this.ValidateMeasureTime(measure),
+                  query_footer2: this.GetDisplayMeasureTime(measure),
```

Comment:
`footer2` was created so that we can put in a nicely formatted version of the time.